ext/hal/nordic/nrfx/hal/nrf_clock.h

```diff
@@ -48,9 +48,8 @@ extern "C" {
  * (HFCLK) settings.
  */
 
-#if defined(NRF52810_XXAA) || \
-    defined(NRF52832_XXAA) || defined(NRF52832_XXAB) || \
-    defined(NRF52840_XXAA)
+#if defined(CLOCK_LFCLKSRC_BYPASS_Enabled) && \
+    defined(CLOCK_LFCLKSRC_EXTERNAL_Enabled)
```

Comment:
Alternatively you could just add the NRF52811 in the list. But I am OK if this is effectively identical.

Reply:
imo, it's more future proof